cmd/buildah/images.go

```diff
@@ -24,22 +46,35 @@ var (
 			Usage: "do not print column headings",
 		},
 		cli.BoolFlag{
-			Name:  "notruncate",
+			Name:  "no-trunc, notruncate",
```

Comment:
I hadn't looked at any of the other commands.  When I wrote this code for kpod, it was supposed to match the docker cli, which uses `--no-trunc`.  I chose not to remove the option when I ported the code to buildah.  I can remove it or add the option to the other commands if you would like